README.rst

```diff
@@ -19,36 +19,34 @@
 .. |Binder| image:: https://mybinder.org/badge_logo.svg
    :target: https://mybinder.org/v2/gh/sunpy/sunpy/main?filepath=examples
 
-SunPy is an open-source Python library for Solar Physics data analysis and visualization.
-Our homepage `SunPy`_ has more information about the project.
+``sunpy`` is an open-source Python library for Solar Physics data analysis and visualization.
+`Our homepage has more information about the SunPy project. <https://sunpy.org>`_
```

Comment:
Making the whole sentence a link is clunky.  Once you mention the "SunPy ecosystem" above, you can just link that phrase.

Reply:
Fixed hopefully.